### ISSUE TITLE
Kryptering malmo.se

### ISSUE THREAD

**Issue by Svempan** [malmostad/sitevision_assets#360]
Vi bör införa https på hela malmo.se
Dels för att vi har sidor med känslig info som inte bör avlyssnas dels för att vi blir bättre rankade av Google

**Comment by magnivar** [comment]
Vi ska införa https på hela malmo.se
Dels för att vi har sidor med känslig info som inte bör avlyssnas dels för att vi blir bättre rankade av Google

Vi tar fram en handlingplan tillsammans med @EllinorHallett och ITS

**Comment by EllinorHallett** [comment]
Detta ska inte vara något problem. Följa steg för steg guiden enligt: [https://help.sitevision.se/SiteVision_4_0/12703972.html](https://help.sitevision.se/SiteVision_4_0/12703972.html)

I stora drag:
- Installera certifikat
- Starta om SiteVision
- Aktivera åtkomstrav

Enligt SiteVision bör det gå utan problem.

**Comment by badwiz** [comment]
Malmö stad behöver beställa ett certifikat på formatet PKCS12 som ska gälla för minst www.malmo.se och malmo.se. Vi rekommenderar ett * certifikat då fungerar *.malmo.se. 
Installationen beräknas inte ta lång tid, utan det handlar om att ha tillgång till filsystemet på SiteVision-servern och ändra 3 parametrar, samt ladda upp filen på server. Detta gör ingen skillnad i anropen. Dock måste tjänsten startas om när konfigurationen är gjord.
Efter omstart måste en admin gå in i SiteVision ooch testa https på någon sida i strukturen för att verifiera att det fungerar som det ska. Sen slår man på https på hela webbplatsen.

**Comment by magnivar** [comment]
Wildcard certifikat finns redan på servern.
magnus kollar med @Svempan  vad som gick snett senast https infördes.

**Comment by badwiz** [comment]
https verkar vara påslaget, men bara som "Inget krav". Det fungerar alltså både med http och https.

**Comment by badwiz** [comment]
Menyn verkar "hårdkoda" länkarna till http. Detta måste ses över för att https ska fungera överallt.

**Comment by magnivar** [comment]
@martent har förberett menyn för övergång till https och dessutom gjort en checklista som jag mailat. Han inväntar produktionssättning.

**Comment by henrikekelof** [comment]
Har åtgärdat allt som jag ser att vi kan åtgärda och markerat i PDF:en (länk nedan).

Ytterligare kommentarer:

- Nya spellisteelementet åtgärdar två av punkterna, men det är bara importerat till Prod, ej Komin. Gamla portlets i sidor är inte heller utbytta. Det går bra att göra det när som helst, men är det vi som ska ta den pucken eller ska ni göra det?

- Länken "Fler filmer" i elementet går till http: för video.malmo.se. Det är ganska OK eftersom det kommer att bli en redirect där, men den kan hur som helst inte ändras innan video-webben fungerar med https.

[https_malmo.se.pdf](https://github.com/malmostad/sitevision_assets/files/1635113/https_malmo.se.pdf)

**Comment by magnivar** [comment]
@henrikekelof Jag är osäker på om spellisteportleten används överhuvudtaget på komin. Kan man kolla det på något sätt? Annars är det ju bara att lägga till det elementet även på Komin (PROD).

**Comment by henrikekelof** [comment]
@magnivar Jo, den finns på några ställen. Sidan Vår kommun och flera av förvaltningarnas startsidor. Men det är inget som hindrar heller. Vill du att jag lägger ut elementet eller ska du?
